debian/pbspro-execution.prerm

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# postrm script for pbspro
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postrm> `remove'
+#        * <postrm> `purge'
+#        * <old-postrm> `upgrade' <new-version>
+#        * <new-postrm> `failed-upgrade' <old-version>
+#        * <new-postrm> `abort-install'
+#        * <new-postrm> `abort-install' <old-version>
+#        * <new-postrm> `abort-upgrade' <old-version>
+#        * <disappearer's-postrm> `disappear' <overwriter>
+#          <overwriter-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    purge|remove|upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
+    if [ -x /sbin/chkconfig -a -x /sbin/service ]; then
+        out=`/sbin/chkconfig --list pbs 2>/dev/null`
+        if [ $? -eq 0 ]; then
+            if [ -x /etc/init.d/pbs ]; then
+                    /etc/init.d/pbs stop
+            else
+                    /sbin/service pbs stop
+            fi
```

Comment:
@prakashcv13 is finding himself in a situation I'm all too familiar with. If you see something that could be improved, but isn't broken, do you fix it? If you do, you run the risk of breaking something else. If we do change it here, then we should also change it in the %preun section of pbspro.spec. And if we change it in pbspro.spec, then we should also change it in pbspro-extensions/packages/linux/pbs.spec.

At the end of the day, we need to consolidate our commercial and OSS spec files. This will likely involve changes to the way we build and test the commercial release. So this simple change is really just the tip of a much larger iceberg.